src/TSheetsRestClient.php

```diff
@@ -1,4 +1,7 @@
 <?php
+
+namespace TSheets;
+
```

Comment:
If we add the line use Exception; here, then we don't need to explicitly set the FQN of the Exception at the TSheetsException class declaration. And it'll fix a number of other potential errors when we are trying to throw new Exception in other code in this file (rather than throw new \Exception()). Or we could just change the TSheetsRestClient constructor to throw a TSheetsException rather than just throwing Exception. Otherwise that one in the constructor would need to be throw new \Exception as well.
```suggestion
use Exception;
```